common/build/eslint-config-fluid/minimal.js

```diff
@@ -95,6 +95,7 @@ module.exports = {
                 "ignoreEOLComments": true
             }
         ],
+        "use-isnan": "error",
```

Comment:
Were there no offenses in the repo? 😮

Reply:
Nope :) This one was easy.

Reply:
I confirmed that my environment was correctly detecting issues by doing #10365.